agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.cpp

```diff
@@ -2539,22 +2522,10 @@ bool backhaul_manager::handle_client_capability_query(ieee1905_1::CmduMessageRx
 
     auto db = AgentDB::get();
 
-    //Check if it is an error scenario - if the STA specified in the Client Capability Query message is not associated
-    //with any of the BSS operated by the Multi-AP Agent [ though the TLV does contain a BSSID, the specification
-    // says that we should answer if the client is associated with any BSS on this agent.]
-    bool associated_client_found = false;
-    sClientInfo client_info;
-    for (const auto &slave : slaves_sockets) {
-        auto associated_clients_map = slave->associated_clients_map;
-        for (const auto &vap : associated_clients_map) {
-            auto it = vap.second.find(client_info_tlv_r->client_mac());
-            if (it != vap.second.end()) {
-                associated_client_found = true;
-                client_info             = it->second;
-                break;
-            }
-        }
-    }
+    // Check if it is an error scenario - if the STA specified in the Client Capability Query
+    // message is not associated with any of the BSS operated by the Multi-AP Agent [ though the
+    // TLV does contain a BSSID, the specification says that we should answer if the client is
+    // associated with any BSS on this agent.]
```

Comment:
this comment should probably be moved to the new location where the check is actually performed couple of lines down this function

Reply:
moved